.azure-pipelines/gpu-tests.yml

```diff
@@ -51,9 +51,9 @@ jobs:
       displayName: 'Image info & NVIDIA'
 
     - bash: |
-        python -c "fname = 'requirements/extra.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
-        pip install "fairscale>=0.4.5"
-        pip install "deepspeed<0.6.0"  # https://github.com/microsoft/DeepSpeed/issues/1878
+        pip install fire
+        python -c "fname = 'requirements/strategies.txt' ; lines = [line for line in open(fname).readlines() if 'horovod' not in line] ; open(fname, 'w').writelines(lines)"
+        pip install "deepspeed<0.6.0"  # TODO: Temporarily use 0.6.0 due to https://github.com/microsoft/DeepSpeed/issues/1878
```

Comment:
this deepspeed reinstallation can be removed as https://github.com/microsoft/DeepSpeed/issues/1878 is now resolved in the latest release 0.6.3.
```suggestion
```

Reply:
Tried 0.6.3 in this CI run https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/results?buildId=69283&view=logs&j=3afc50db-e620-5b81-6016-870a6976ad29, and there seems no problem.

Reply:
setting `deepspeed>=0.6.3`